angular-primeng-app/src/app/app.component.ts

```diff
@@ -1,17 +1,18 @@
 import { Component, OnInit, OnDestroy, inject, Inject } from "@angular/core";
 import { RouterOutlet } from "@angular/router";
-import { Subscription } from "rxjs";
+import { Subscription, fromEvent } from "rxjs";
 import { BlogInfo } from "./models/blog-info";
 import { BlogService } from "./services/blog.service";
 import { ThemeService } from "./services/theme.service";
-import { DOCUMENT } from "@angular/common";
+import { DOCUMENT, ViewportScroller } from "@angular/common";
 import { HeaderComponent } from "./components/header/header.component";
 import { FooterComponent } from "./components/footer/footer.component";
+import { ButtonModule } from "primeng/button";
 
 @Component({
 	selector: "app-root",
 	standalone: true,
-	imports: [RouterOutlet, HeaderComponent, FooterComponent],
+	imports: [RouterOutlet, HeaderComponent, FooterComponent,ButtonModule],
```

Comment:
@DongreJaipal could you add a space before `ButtonModule` so there is a space between the comma and the button module like this 
`imports: [RouterOutlet, HeaderComponent, FooterComponent, ButtonModule],`